coremltools/converters/mil/mil/ops/defs/conv.py

```diff
@@ -22,8 +22,7 @@
 @register_op(doc_str="")
 class conv(Operation):
     """
-    Perform convolution over input. Currently supports only 1-D and 2-D
```

Comment:
Please make a similar change for the doc string of `class conv_transpose(Operation):` as well, which also supports 3-D conv transpose, but the doc string does not say that (incorrectly)

Reply:
OK